src/openzaak/utils/__init__.py

```diff
@@ -15,12 +17,43 @@ def parse_isodatetime(val) -> datetime:
     return dateutil.parser.parse(val)
 
 
-def build_absolute_url(path: str, request: Optional[HttpRequest] = None) -> str:
+def get_openzaak_domain() -> str:
+    """
+    Obtain the domain/netloc of Open Zaak according to settings or configuration.
+    """
     from django.contrib.sites.models import Site
 
+    if settings.OPENZAAK_DOMAIN:
+        return settings.OPENZAAK_DOMAIN
+
+    warnings.warn(
```

Comment:
Hrm no, not at this time - this is now basically a centralized location to obtain the Open Zaak domain and it warns when it's going to fallback behaviour. When we've refactored all the usage of contrib.sites as a first option in favour of the new setting, we can make this a `DeprecationWarning` instead of `PendingDeprecationWarning` and then in the next release we can remove it.